apps/api/middleware/jwt_auth.py

```
@@ -45,23 +61,86 @@ class AuthClaims(TypedDict, total=False):
 JWT_ALGORITHM: Final[str] = "HS256"
+MIN_SECRET_LENGTH: Final[int] = 32  # 256 bits for HS256
 
 # HTTP Bearer token scheme
 bearer_scheme = HTTPBearer(auto_error=False)
 
 
+def _ensure_env_loaded() -> None:
+    """Ensure .env configuration is loaded before reading process environment."""
+    try:
+        from packages.common.config import ensure_env_loaded
+    except Exception as exc:  # pragma: no cover - defensive logging
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)